applist.csv

```diff
@@ -45,8 +46,10 @@ goland,com.jetbrains.GoLand
 granatier,org.kde.granatier
 gwenview,org.kde.gwenview
 inkscape,org.inkscape.Inkscape
+insomnia,rest.insomnia.Insomnia
 intellij-idea-community,com.jetbrains.IntelliJ-IDEA-Community
 intellij-idea-ultimate,com.jetbrains.IntelliJ-IDEA-Ultimate
+joplin,net.cozic.joplin_desktop
```

Comment:
I think this is replacing the terminal version with the desktop app: the desktop app snap that matches this flatpak seems to me to be
joplin-desktop

Reply:
indeed, I got it installed and didn't realise that it was a cli version
will fix, thanks